content/get-haiku/r1beta4/_index.md

```diff
@@ -0,0 +1,165 @@
+</ul>
+</div>
+
+<!--
+Disabled until we move away from R1/beta4
+{{< alert-info "Nightly Images" "Looking for the Nightly Images? They can be found at download.haiku-os.org.">}}
+-->
+
+## Download a Haiku Image
+
+<i>If you have an earlier version of Haiku installed, [see upgrade instructions below](#upgrading-from-r1beta3).</i>
+
+<p>The provided images can be written directly to a <a href="/guides/installing/making_haiku_usb_stick">USB flash drive</a>, an empty disk, or <a href="/get-haiku/burn-cd">written to DVD</a> media.<br /> You can boot directly from your chosen physical media and try out Haiku without needing to install it.</p>
+The images can be used "live" or used to install Haiku to another disk/partition of your choice.<br />
+<p>
+They can also be used in <a href="/guides/virtualizing">virtual machines</a> such as QEMU, VMWare or VirtualBox.
+</p>
+
+### Direct Download Locations
+
+If you choose to download an image, please select from the list of available mirrors below. Mirrors are in no specific order.
+
+{{< alert-info "Not all mirrors may be online" "Please choose another mirror if you encounter a failure. It is recommended you choose the mirror closest to your geographical location.">}}
+
+<div class="nolinks">
+<table id="mirrors" class="table table-hover">
+<thead style="font-weight: bold;">
+<tr>
+<td>Mirror</td>
+<td title="BeOS API + binary compatible">32-bit</td>
+<td title="BeOS API compatible">64-bit</td>
+</tr>
+</thead>
+<tbody>
+    <tr class="link">
+      <td class="location"><b>Location:</b> New York, United States <br/>Provided by: <a target="_blank" class="ext" href="http://www.rit.edu" title="RIT">Rochester Institute of Technology</a></td>
+      <td><a class="track" href="http://mirror.rit.edu/haiku/r1beta4/haiku-r1beta4-x86_gcc2h-anyboot.iso">iso</a></td>
+      <td><a class="track" href="http://mirror.rit.edu/haiku/r1beta4/haiku-r1beta4-x86_64-anyboot.iso">iso</a></td>
+    </tr>
+    <tr class="link">
+      <td class="location"><b>Location:</b> Oregon, United States <br/>Provided by: <a target="_blank" class="ext" href="http://www.osuosl.org" title="OSUOSL">Oregon State University</a></td>
+      <td><a class="track" href="https://ftp.osuosl.org/pub/haiku/r1beta4/haiku-r1beta4-x86_gcc2h-anyboot.iso">iso</a></td>
+      <td><a class="track" href="https://ftp.osuosl.org/pub/haiku/r1beta4/haiku-r1beta4-x86_64-anyboot.iso">iso</a></td>
+    </tr>
+    <tr class="link">
+      <td class="location"><b>Location:</b> North Virginia, United States<br/>Provided by: <a target="_blank" class="ext" href="https://haiku-inc.org" title="Haiku, Inc.">Haiku, Inc.</a></td>
+      <td><a class="track" href="https://s3.us-east-1.wasabisys.com/haiku-release/r1beta4/haiku-r1beta4-x86_gcc2h-anyboot.iso">iso</a></td>
+      <td><a class="track" href="https://s3.us-east-1.wasabisys.com/haiku-release/r1beta4/haiku-r1beta4-x86_64-anyboot.iso">iso</a></td>
+    </tr>
+    <tr class="link">
```

Comment:
I  think we should comment this mirror out.

It kinda looks like they were mirroring IPFS (which is fine), but it also looks like someone was moving files around by hand.  (I never moved r1/beta4 into the release directory for r1beta4)